packages/backend-core/src/platform/users.ts

```diff
@@ -113,15 +113,12 @@ export async function addUser(
 export async function removeUser(user: User) {
   const db = getPlatformDB()
   const keys = [user._id!, user.email]
-  const userDocs = await db.allDocs({
+  const userDocs = await db.allDocs<User>({
     keys,
     include_docs: true,
   })
-  const toDelete = userDocs.rows.map((row: any) => {
-    return {
-      ...row.doc,
-      _deleted: true,
-    }
-  })
-  await db.bulkDocs(toDelete)
+  await db.bulkRemove(
+    userDocs.rows.map(row => row.doc!),
+    { silenceErrors: true }
```

Comment:
Why do we want to silence errors? Can't we have issues with concurrent update & delete, or similar?

Reply:
This is just for consistency with how we used to use this, previously when we used this without a utility errors would be ignored, I wanted to use this in a few places as it made the typing better, but I didn't want to introduce a new failure point, some of these APIs are years old and have been ignoring errors in deletion for all that time, I didn't want to change the behaviour.